samples/Meeting/src/app/utils/getToken.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export type UserToken = {
+  expiresIn: number;
+  identity: string;
+  token: string;
+};
+
+// TODO: Need to revisit if cachedUserToken with respect to auto refresh token
+let cachedUserToken: UserToken | undefined = undefined;
```

Comment:
likewise with endpointUrl, does this need to be cached in a global variable? (is getToken called a lot?)

Reply:
this file isn't actually needed it would seem its actually using the calls in the utils.ts file to get the token. will remove.